mne/io/snirf/tests/test_snirf.py

```diff
@@ -288,6 +296,29 @@ def test_snirf_nirsport2_w_positions():
     assert len(mon.dig) == 43
 
 
+@requires_testing_data
+@requires_h5py
+def test_snirf_kernel_hb():
+    """Test reading Kernel SNIRF files with haemoglobin data."""
+    raw = read_raw_snirf(kernel_hb, preload=True)
+
+    # Test data import
+    assert raw._data.shape == (180 * 2, 14)
+    assert raw.copy().pick('hbo')._data.shape == (180, 14)
+    assert raw.copy().pick('hbr')._data.shape == (180, 14)
+
+    assert_almost_equal(raw.info['sfreq'], 8.256, decimal=2)
```

Comment:
Can you add a comment why we only get two decimals? That seems pretty bad. If possible assert_allclose with rtol=1e-6 or something would be better (single float tolerance territory)

Reply:
I think the person who gave me the file just wasn't being accurate when they conveyed the values to me. As such, I have changed the test to use the more accurate value in the data.